easybuild/easyconfigs/l/LAMMPS/LAMMPS-7Aug2019-intel-2019b-Python-3.7.4-kokkos-OCTP.eb

```diff
@@ -0,0 +1,112 @@
+name = 'LAMMPS'
+version = '7Aug2019'
+local_python_versionsuffix = '-Python-%(pyver)s'
+versionsuffix = local_python_versionsuffix
+versionsuffix += '-kokkos-OCTP'
+
+homepage = 'https://lammps.sandia.gov/'
+description = """LAMMPS is a classical molecular dynamics code, and an acronym
+for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has
+potentials for solid-state materials (metals, semiconductors) and soft matter
+(biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be
+used to model atoms or, more generically, as a parallel particle simulator at
+the atomic, meso, or continuum scale. LAMMPS runs on single processors or in
+parallel using message-passing techniques and a spatial-decomposition of the
+simulation domain. The code is designed to be easy to modify or extend with new
+functionality.
+"""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'openmp': True, 'usempi': True, 'cstd': 'c++11'}
+
+source_urls = [
+    # 'https://github.com/lammps/lammps/archive/'
```

Comment:
Is it really useful to leave this here? We also don't 'explain'  in a comment what `GITHUB_LOWER_SOURCE` is in other EC's, right?